src/pages/bounties/[id]/[[...slug]].tsx

```diff
@@ -0,0 +1,571 @@
+  }, [bounty, currentUser]);
+  const currency = getBountyCurrency(bounty);
+
+  const minUnitAmount = bounty.minBenefactorUnitAmount;
+  const Icon = CurrencyConfig[currency].icon;
+
+  const isFavorite = false;
+  const onFavoriteClick = () => {
+    console.log('is favorite');
+  };
+
+  const onAddToBounty = (amount: number) => {
+    addBenefactorUnitAmountMutation({ bountyId: bounty.id, unitAmount: amount });
+  };
+
+  const bountyDetails: DescriptionTableProps['items'] = [
+    {
+      label: 'Bounty Type',
+      value: (
+        <Group spacing={0} noWrap position="apart">
+          <Badge radius="xl" color="gray">
+            {getDisplayName(bounty.type)}
+          </Badge>
+        </Group>
+      ),
+    },
+    {
+      label: 'Bounty Mode',
+      value: (
+        <Group spacing={0} noWrap position="apart">
+          <Badge radius="xl" color="gray">
+            {getDisplayName(bounty.mode)}
+          </Badge>
+        </Group>
+      ),
+    },
+    {
+      label: 'Entry Mode',
+      value: (
+        <Group spacing={0} noWrap position="apart">
+          <Badge radius="xl" color="gray">
+            {getDisplayName(bounty.entryMode)}
+          </Badge>
+        </Group>
+      ),
+    },
+    {
+      label: 'Date started',
+      value: (
+        <Group spacing={0} noWrap position="apart">
+          <Text>{formatDate(bounty.startsAt)}</Text>
+        </Group>
+      ),
+    },
+    {
+      label: 'Deadline',
+      value: (
+        <Group spacing={0} noWrap position="apart">
+          <Text>{formatDate(bounty.expiresAt)}</Text>
+        </Group>
+      ),
+    },
+    {
+      label: 'Tags',
+      value: (
+        <Group spacing={0} noWrap position="apart">
+          <Badge radius="xl" color="gray">
+            TODO.bounties
+          </Badge>
+        </Group>
+      ),
+    },
+  ];
+
+  const benefactorDetails: DescriptionTableProps['items'] = bounty.benefactors.map((b) => ({
+    label: (
+      <Group spacing={4}>
+        <UserAvatar user={b.user} withUsername linkToProfile />
+        {isMainBenefactor(bounty, b.user) && (
+          <IconStar
+            color={CurrencyConfig[currency].color(theme)}
+            fill={CurrencyConfig[currency].color(theme)}
+            size={12}
+          />
+        )}
+      </Group>
+    ),
+    value: (
+      <Group spacing={4} style={{ float: 'right' }}>
+        <Icon
+          color={CurrencyConfig[currency].color(theme)}
+          fill={CurrencyConfig[currency].color(theme)}
+          size={20}
+        />
+        <Text weight={590}>{formatCurrencyForDisplay(b.unitAmount, currency)}</Text>
+      </Group>
+    ),
+  }));
+
+  const files = bounty.files ?? [];
+  const filesCount = files.length;
+
+  return (
+    <Stack>
+      <Group>
+        {addToBountyEnabled && (
+          <Group color="gray" p={4} style={{ background: theme.colors.dark[6] }}>
+            <Group spacing={2}>
```

Comment:
I have a feeling that one of these `<Group>` needs a `noWrap`, not sure which one 😛